frontend_app_kanban/src/component/KanbanCardEditor.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import { IconButton } from 'tracim_frontend_lib'
+
+function KanbanCardEditor (props) {
+  const { card } = props
+
+  const [title, setTitle] = React.useState(card.title || '')
+  const [description, setDescription] = React.useState(card.description || '')
+  const [colorEnabled, setColorEnabled] = React.useState(!!card.bgColor)
+  const [bgColor, setBgColor] = React.useState(card.bgColor || '')
+  const [deadline, setDeadline] = React.useState(card.deadline || '')
+  const [duration, setDuration] = React.useState(card.duration || '')
+  const [htmlEnabledForDescription, setHtmlEnabledForDescription] = React.useState(!!card.htmlEnabledForDescription)
+
+  function handleValidate (e) {
+    e.preventDefault()
+    props.onValidate({
+      ...card,
+      title,
+      description,
+      htmlEnabledForDescription,
+      bgColor: colorEnabled ? bgColor : '',
+      deadline,
+      duration
+    })
+  }
+
+  const editorTitle = props.card.id ? props.t('Editing Card') : props.t('New Card')
+  return (
+    <div className='file__contentpage__statewrapper__kanban__KanbanCardEditor'>
+      <div className='file__contentpage__statewrapper__kanban__KanbanCardEditor__title'>
+        <span className='file__contentpage__statewrapper__kanban__KanbanCardEditor__title__icon'>
+          <i
+            className='fa fa-id-card-o'
+            title={editorTitle}
+          />
+        </span>
+        <span className='file__contentpage__statewrapper__kanban__KanbanCardEditor__title__name'>
+          {editorTitle}
+        </span>
+      </div>
+      <form className='file__contentpage__statewrapper__kanban__KanbanCardEditor__form' onSubmit={handleValidate}>
+        <div className='file__contentpage__statewrapper__kanban__KanbanCardEditor__form__fields'>
+          <p>
+            <span>
+              <label htmlFor='file__contentpage__statewrapper__kanban__KanbanCardEditor__title'>{props.t('Title:') + ' '}</label>
+            </span>
+            <span>
+              <input autoFocus={!props.focusOnDescription} id='file__contentpage__statewrapper__kanban__KanbanCardEditor__title' type='text' value={title} onChange={(e) => setTitle(e.target.value)} />
+            </span>
+          </p>
+          <p>
+            <span>
+              <label htmlFor='file__contentpage__statewrapper__kanban__KanbanCardEditor__description'>{props.t('Description:') + ' '}</label>
+            </span>
+            <span>
+              <textarea autoFocus={props.focusOnDescription} id='file__contentpage__statewrapper__kanban__KanbanCardEditor__description' value={description || ''} onChange={(e) => setDescription(e.target.value)} />
+              <br />
+              <label><input type='checkbox' checked={htmlEnabledForDescription} onChange={(e) => setHtmlEnabledForDescription(e.target.checked)} />{props.t('Enable HTML')}</label>
+            </span>
```

Comment:
Textarea probably should not be a child of span, since it is a block element: https://developer.mozilla.org/fr/docs/Web/HTML/Element/Textarea